torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -273,16 +292,144 @@ void PyTorchModelLoader::loadConvolution(const torch::jit::Node *ptNode) {
       f_->createConv("conv", input, weights, bias, outTy, kernels, strides,
                      pads, groups, dilation);
   glow::TransposeNode *output = f_->createTranspose(
-      "conv_input_transposed", conv->getResult(), NHWC2NCHW);
+      "conv_output_transposed", conv->getResult(), NHWC2NCHW);
 
   addGlowNodeValue(outputs[0], output->getResult());
 }
 
+void PyTorchModelLoader::loadBatchNorm(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  assert(inputs.size() == 9);
```

Comment:
DCHECK everywhere?

Reply:
I will go through in a follow up and change things to DCHECK or llvm::Error now that things are more settled.